pkg/process/checks/net.go

```diff
@@ -106,8 +106,9 @@ func (c *ConnectionsCheck) getConnections() ([]*model.Connection, error) {
 		}
 		cs, err := c.localTracer.GetActiveConnections(c.tracerClientID)
 		conns := make([]*model.Connection, len(cs.Conns))
+		cache := make(ebpf.AddrCache)
```

Comment:
This is kinda of ugly (should be hidden from clients) but once we get rid of the local system-probe bth `FormatConnection` and `AddrCache` will become private and be called internally from the marshalers